src/cls/_ZPM/Storage/ModuleTemplate.cls

```diff
@@ -162,7 +162,7 @@ ClassMethod GetCSPApplications(Output pApplications As %List, currentNamespace =
 }
 
 /// adds a web application to the temporary array of module resources
-Method AddWebApps(pApps As %String, Output cspapps As %List, curns = {$Namespace}) As %Status
+Method AddWebApps(pApps As %String, Output cspapps As %List) As %Status
```

Comment:
Again no need to remove the argument. Leave as is so namespace can be overridden